toolkits/nature/packages/nature-sort-by/js/sort-by.js

```diff
@@ -0,0 +1,41 @@
+import {Expander} from '@springernature/global-expander/js/expander';
+import {makeArray} from '@springernature/global-javascript/src/helpers';
+
+const generateParameters = value => {
+	const parameters = new URL(document.location).searchParams;
+	parameters.set('order', value);
+	return parameters;
+};
+
+const sortBy = () => {
+	const trigger = document.querySelector('[data-sort-by-trigger]');
+	const targetElement = document.querySelector('[data-sort-by-target]');
+	const radios = makeArray(document.querySelectorAll('[data-sort-by-radio]'));
```

Comment:
Can you detail the data attributes needed for this to work in the readme, and what they are used for